conf/provisioning/dashboards/nuodb/disk.json

```diff
@@ -1956,12 +2201,58 @@
               "type": "fill"
             }
           ],
+          "hide": true,
+          "measurement": "WriteThrottleTime",
+          "orderByTime": "ASC",
+          "policy": "default",
+          "query": "SELECT mean(B) FROM (SELECT mean(\"value\") AS B FROM \"WriteThrottleTime\" WHERE (\"nodetype\" = 'Transaction') AND \"db\" =~ /^$database$/ AND $timeFilter GROUP BY time($__interval) fill(null)) WHERE $timeFilter GROUP BY time($__interval) ",
+          "rawQuery": false,
+          "refId": "D",
+          "resultFormat": "time_series",
+          "select": [
+            [
+              {
+                "params": [
+                  "value"
+                ],
+                "type": "field"
+              },
+              {
+                "params": [],
+                "type": "max"
+              }
+            ]
+          ],
+          "tags": [
+            {
+              "key": "nodetype",
+              "operator": "=",
+              "value": "Transaction"
+            }
+          ]
+        },
+        {
+          "alias": "$tag_host($tag_db:$tag_nodeid):queuedepth",
```

Comment:
Can we use the same alias as in the other panels? I.e $tag_host(db: $tag_db, nodeid: $tag_nodeid, startid: $tag_startid).

Reply:
yes this should be changed.